packages/patrol/example/integration_test/open_app_test.dart

```diff
@@ -5,14 +5,14 @@ import 'package:flutter/material.dart';
 import 'common.dart';
 
 void main() {
-  late String mapsId;
-  if (Platform.isIOS) {
-    mapsId = 'com.apple.Maps';
-  } else if (Platform.isAndroid) {
-    mapsId = 'com.google.android.apps.maps';
-  }
-
-  patrol('counter state is the same after switching apps', ($) async {
+  patrol('open maps', ($) async {
+    late String mapsId;
+    if (Platform.isIOS) {
+      mapsId = 'com.apple.Maps';
+    } else if (Platform.isAndroid) {
+      mapsId = 'com.google.android.apps.maps';
+    }
```

Comment:
What happens in the else case? mapsId is not initialized?

Reply:
If we get to `else` it means it's neither Android nor iOS, so we don't care.

Reply:
Then i would recommend not marking it as late (hides the problem) and instead add an else branch where you throw some error.

Reply:
right

done